### ISSUE TITLE
About the readability

### ISSUE THREAD

**Issue by familyld** [aviralkumar2907/CQL#3]
Hi Aviral,

In the paper, you claim CQL can be implemented with less than 20 lines of code, but it's really difficult to identify these "20 lines of code" from the current version of your project which is built upon other projects. Would you please point out which part of code exactly corresponds to the core of CQL? I really like your idea of CQL, both the theoretical part and its simplicity, but currently, it seems very hard to follow. 

Best,
Zhi-Hong

**Comment by familyld** [comment]
For the Atari experiments, it seems like line [221](https://github.com/aviralkumar2907/CQL/blob/d67dbe9cf5d2b96e3b462b6146f249b3d6569796/atari/batch_rl/multi_head/quantile_agent.py#L221) ~ [243](https://github.com/aviralkumar2907/CQL/blob/d67dbe9cf5d2b96e3b462b6146f249b3d6569796/atari/batch_rl/multi_head/quantile_agent.py#L243) in quantile_agent.py are of interest.

For the D4RL experiments, it seems like line [233](https://github.com/aviralkumar2907/CQL/blob/d67dbe9cf5d2b96e3b462b6146f249b3d6569796/d4rl/rlkit/torch/sac/cql.py#L233) ~ [300](https://github.com/aviralkumar2907/CQL/blob/d67dbe9cf5d2b96e3b462b6146f249b3d6569796/d4rl/rlkit/torch/sac/cql.py#L233) in cql.py are of interest, but more lines of code are modified or added, comparing to sac.py.

Since the two experiments are based on different methods (QR-DQN and SAC) and different libraries (dopamine and rlkit),  the detailed implementations are quite different. It would be great if the author can add more comments and explicitly correspond the code to the equations in the paper, e..g., Equation (4).

**Comment by aviralkumar2907** [comment]
Hello @familyld,

Yes, I hope to add some more comments in about a month timeline. Sorry for the delay in doing that.
